calculatorBackend/src/Rest/src/Rest.cpp

```diff
@@ -0,0 +1,47 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   main.cpp
+ *  @brief  Entry Point of the rest implementation
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include "Rest.h"
+
+#include <iostream>
+
+#include "pistache/endpoint.h"
+#include "pistache/router.h"
+
+#include "Calculator.h"
+#include "CalculatorMessages.h"
```

Comment:
I don't believe Rest needs Parser.h or CalculatorMessages.h, also including just rest.h takes into account pistache/endpoint and /router. This will speedup build time

Reply:
I did end up using messages in here, but yea parser wasn't needed for sure